tests/testnets/simple.common/01-gen-configs.sh

```diff
@@ -149,10 +148,12 @@ datadir = \${sbws_home}/datadir
 sbws_home = $(pwd)
 
 [tor]
-control_type = socket
-control_location = \${paths:sbws_home}/$scanner_tor_socks_proxy_nick/control_socket
-socks_host = $scanner_tor_socks_proxy_ip
-socks_port = $socksport
+extra_lines =
+    TestingTorNetwork 1
+    NumCPUs 1
+    LogTimeGranularity 1
+    SafeLogging 0
```

Comment:
Consider UseEntryGuards 0 to disable pathbias warnings.
See #146.

Reply:
@teor2345 [I don't think we should use FetchDirInfo*Early](https://github.com/pastly/simple-bw-scanner/issues/146#issuecomment-385976197)

Also, notice the inclusion of them in initial commits on this branch is undone by latter commits (ce2187e).

Reply:
I did not notice, I was using the files view.

Reply:
@juga0 these lines are *in addition* to the torrc config lines you linked to. See how in [b378bf63](https://github.com/pastly/simple-bw-scanner/pull/147/commits/b378bf63e09fcbc3a060c1d387363975f89e29b9#diff-a250492b468dd26963e8a1c2343cb1e1R175) I added the ability for torrc options to be read from config.ini.

So in the test networks, all the options in `sbws/util/stem.py launch_tor()` are set, and then these `extra_lines` are added.

Does that make sense? I'm not sure I've addressed your concern.